v1/proto/gribi_aft/gribi_aft.proto

```diff
@@ -68,6 +68,66 @@ message Afts {
+        ywrapper.StringValue dst_ip = 469869578 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/ipv6/state/dst-ip"];
+        ywrapper.StringValue src_ip = 254664891 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/ipv6/state/src-ip"];
+      }
+      message Mpls {
+        message MplsLabelStackUnion {
+          gribi_aft.enums.OpenconfigMplsTypesMplsLabelEnum mpls_label_stack_openconfigmplstypesmplslabelenum = 212167609 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/mpls-label-stack"];
+          uint64 mpls_label_stack_uint64 = 205621201 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/mpls-label-stack"];
+        }
+        repeated MplsLabelStackUnion mpls_label_stack = 310218456 [(yext.leaflistunion) = true,(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/mpls-label-stack"];
+        ywrapper.UintValue traffic_class = 380564126 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/traffic-class"];
+      }
+      message UdpV4 {
+        ywrapper.UintValue dscp = 404304264 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/dscp"];
+        ywrapper.StringValue dst_ip = 425782075 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/dst-ip"];
+        ywrapper.UintValue dst_udp_port = 26930775 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/dst-udp-port"];
+        ywrapper.UintValue ip_ttl = 489501006 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/ip-ttl"];
+        ywrapper.StringValue src_ip = 223532242 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/src-ip"];
+        ywrapper.UintValue src_udp_port = 332943154 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v4/state/src-udp-port"];
+      }
+      message UdpV6 {
+        ywrapper.UintValue dscp = 493128346 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dscp"];
+        ywrapper.StringValue dst_ip = 148029921 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-ip"];
+        ywrapper.UintValue dst_udp_port = 275733981 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-udp-port"];
+        ywrapper.UintValue ip_ttl = 394480800 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/ip-ttl"];
+        ywrapper.StringValue src_ip = 494881688 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-ip"];
+        ywrapper.UintValue src_udp_port = 485812424 [(yext.schemapath) = "/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-udp-port"];
+      }
+      message Vxlan {
```

Comment:
Should we `deviate` this out? I'm not sure it's supported at all via gRIBI and perhaps we don't want to give the impression it is?